src/@types/svg.d.ts

```diff
@@ -0,0 +1,9 @@
+// https://github.com/facebook/create-react-app/blob/0ee4765c39f820e5f4820abf4bf2e47b3324da7f/packages/react-scripts/lib/react-app.d.ts#L47-L56
+
+declare module '*.svg' {
+  import * as React from 'react'
+
+  export const ReactComponent: React.FunctionComponent<
+    React.SVGProps<SVGSVGElement> & { title?: string }
+  >
+}
```

Comment:
not sure what it does exactly - adding types to svg?

edit: ok, this is so that we can import svg to typescript

Reply:
yes, but I'm not sure if this works as desired after I've updated all of the packages to the latest versions... I'll look into that